### DISCUSSION TITLE
如何自定义连接桩的样式和鼠标放上去才显示

### DISCUSSION THREAD

**Discussion by outian** [antvis/X6#1665] [Q&A] [answered]
类似这种效果，如何实现，麻烦大神们给个思路

![测试.gif](https://s2.loli.net/2021/12/20/yR1H7NTWSPogvXm.gif)

**Comment by NewByVector** [comment, marked as answer]
自定义连接桩可以参考这个 [demo](https://x6.antv.vision/zh/examples/showcase/practices#er),自定义显示可以参考这个 [demo](https://x6.antv.vision/zh/examples/showcase/practices#flowchart)
